tembo-cli/src/main.rs

```diff
@@ -61,6 +63,9 @@ fn main() -> Result<(), anyhow::Error> {
         SubCommands::Delete(_delete_cmd) => {
             delete::execute()?;
         }
+        SubCommands::Logs(logs_cmd) => {
```

Comment:
Likely can remove or set as unused for `logs_cmd`

Reply:
Removed it!